src/mSupplyMobileApp.js

```diff
@@ -68,23 +68,35 @@ class MSupplyMobileAppContainer extends React.Component {
     this.scheduler = new Scheduler();
     const isInitialised = this.synchroniser.isInitialised();
     this.scheduler.schedule(this.synchronise, SYNC_INTERVAL);
+    // this.scheduler.schedule(this.synchroniseSensor, SYNC_INTERVAL);
```

Comment:
can you TODO this please

Reply:
I'd rather make a list of issues rather than chuck TODO's all through if that's all good? I find TODO's all the time, they seem to just get chucked in and forgot about